ford/fortran_project.py

```diff
@@ -28,8 +28,7 @@
 
 import os
 import toposort
-
-import ford.sourceform
```

Comment:
This is a little surprising! Does this module not use `sourceform` directly?

Reply:
I'm sorry might be an oversight from the merging. I am not sure, why it is removed.